docs/tutorials/sessions/createDistributedSession.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_label: "Create a distributed session"
+sidebar_position: 2
+title: "Create a distributed session"
+---
+
+:::info
+Building in React? [check here](../../react/useCreateDistributedSession.md)
+:::
+
+### Overview
+
+This tutorial demonstrates how a dapp can create a distributed session using viem and the Biconomy Smart Account with the `@biconomy/account` SDK. The provided code assumes you have a Biconomy Paymaster API key and a connected user. The following is appropriately viewed from the perspective of a dapp, looking to make txs on a users behalf.
+
+You can get your Biconomy Paymaster API key from the dashboard [here](https://dashboard.biconomy.io/).
+
+
+### What are Distributed Sessions?
+
+Biconomy's [DAN (Delegated Authorisation Network)](https://www.biconomy.io/post/introducing-dan-the-programmable-authorisation-network-for-ai-agents)  is an off-chain authorization network designed to enhance the security, customizability & speed of managing authorization keys for our smart accounts. Our distributed sessions feature leverages DAN (with the economic security of [Eigenlayer AVS](https://docs.eigenlayer.xyz/eigenlayer/overview)) to offer developers a comprehensive, zero-development, zero-custody sessions solution, which can be leveraged directly from your frontend. [Read more about Distributed Sessions here](/Modules/sessions/DistributedSessions)
+
+### Prerequisites
+
+- Biconomy Paymaster API key [If you are using paymaster rules in the dashboard, make sure you have whitelisted session Module contract 0x000002fbffedd9b33f4e7156f2de8d48945e7489]
+- A Bundler url if you don't want to use the testnet one, for Amoy you can use
+
+```
+https://bundler.biconomy.io/api/v2/80002/nJPK7B3ru.dd7f7861-190d-41bd-af80-6877f74b8f44
+```
+
+- A user with a connected signer (viem WalletClient or ethers.Wallet for example)
+
+### Step 1: Create the SmartAccountClient for the user
+
+```typescript
+import { polygonAmoy as chain } from "viem/chains";
+import {
+  PaymasterMode,
+  createSmartAccountClient,
+  createDistributedSession,
+  Rule,
+  Policy,
+  createDistributedSessionKeyEOA,
+} from "@biconomy/account";
+
+const nftAddress = "0x1758f42Af7026fBbB559Dc60EcE0De3ef81f665e";
+
+// Create Biconomy Smart Account instance
+const usersSmartAccount = await createSmartAccountClient({
+  signer: usersWalletClient, // assumes that a user has connected his walletClient (or an ethers Wallet) to your dapp
+  biconomyPaymasterApiKey: config.biconomyPaymasterApiKey,
+  bundlerUrl: config.bundlerUrl,
+});
+const smartAccountAddress = await usersSmartAccount.getAccountAddress();
+```
+
+### Step 2: Create the Policy
```

Comment:
can we mention here that this policy is still not dan permissions checked off-chain but it is for ABI SVM that works with SKM.
and later as part of session key eoa generation, permissions object can be passed that will omit the need for using ABI SVM. maybe under coming soon..

Reply:
Gonna delete this file as it detracts from our core sessions offering. I will instead mention session keys being stored on DAN as an option from the main 'createSession' page.

Reply:
Mentioned on the Distributed Sessions page